roles/edpm_download_cache/tasks/container_images.yml

```diff
@@ -76,3 +76,11 @@
   ansible.builtin.include_role:
     name: osp.edpm.edpm_neutron_dhcp
     tasks_from: download_cache.yml
+
+- name: Download images for edpm_logrotate_crond role
```

Comment:
We already call download_cache.yml during the download of the packages so this task, as it stands, would be redundant. Do you mind splitting the download_cache in two different task files so we can call them independently?